RELEASE.md

```diff
@@ -1,3 +1,37 @@
+# Release 1.7.0
+
+The 1.7 minor series tracks TensorFlow 1.7.
+
+## Hightlights
+
+- (Beta) New Beholder plugin that shows a live video feed of tensor data during
```

Comment:
Please mention this requires a local disk and doesn't work on GCS, S3, etc.

Reply:
Done.